client/app/models/Vcf.iobio.js

```diff
@@ -1467,7 +1467,7 @@ export default function vcfiobio(theGlobalApp) {
   }
 
   exports._parseHeaderForInfoORFormat = function(record) {
-    const matches = record.match(/<ID=(\w+),Number=(\w+),Type=(\w+),Description="([^"]+)">/);
+    const matches = record.match(/<ID=(\w+),Number=(\w+|\.),Type=(\w+),Description="([^"]+)">/);
     if (matches) {
```

Comment:
@YangQi007  There is still a lurking bug that I hinted at in the GitHub issue. What happens when `matches` equal False? What is the value of `infoOrFormat`? It will be set to the last entry. What should it be set to? The expectation of the method caller is that the return value represents the parsed format field when it works and probably null if it doesnt.  I will fix this code after I merge your pull request.

Reply:
@tonydisera yeah, you're right! I should have had a error reporting for the Regular Expression to catch some unexpected record format. Otherwise, we need to debug manually. What is the best practice here? Do you think it makes sense that put a console log here if the match is 'False'?

Reply:
Please review commit [6291bcfb]. This was fixed by setting `infoOrFormat` to null when `matches` = false. Also, the code that calls _parseHeaderForInfoORFormat tests the return value. If null, the field is bypassed (not included in the hash).